app/src/main/java/org/sil/hearthis/ChaptersActivity.java

```diff
@@ -0,0 +1,106 @@
+package org.sil.hearthis;
+
+import Script.BookInfo;
+
+import android.app.Activity;
+import android.content.Context;
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.Gravity;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.AdapterView;
+import android.widget.BaseAdapter;
+import android.widget.GridView;
+import android.widget.TextView;
+
+import org.json.JSONObject;
+
+import java.util.ArrayList;
+
+public class ChaptersActivity extends Activity
```

Comment:
In a shared project, renaming files is a good thing to discuss and coordinate. As you can see, github treats it as adding and deleting the whole file, which makes it hard to identify and review any other changes. So it's best done as an isolated PR where there ARE no other changes (except the required ones where the renamed class is used.)